gwpy/table/gravityspy.py

```diff
@@ -211,18 +213,38 @@ def search(cls, uniqueID, howmany=10):
         from astropy.utils.data import get_readable_fileobj
         import json
         from six.moves.urllib.error import HTTPError
+        from six.moves import urllib
 
         # Need to build the url call for the restful API
         base = 'https://gravityspytools.ciera.northwestern.edu' + \
             '/search/similarity_search_restful_API'
 
+        map_era_to_url = {
+            'ALL': "event_time BETWEEN 1126400000 AND 1229176818",
+            'O1': "event_time BETWEEN 1126400000 AND 1137250000",
+            'ER10': "event_time BETWEEN 1161907217 AND 1164499217",
+            'O2a': "event_time BETWEEN 1164499217 AND 1219276818",
+            'ER13': "event_time BETWEEN 1228838418 AND 1229176818",
+        }
```

Comment:
It would be nice to replace this with an API call that returns the eras, or somehow have the API itself accept an era in the query.

Reply:
I agree with the fact the API should either return a dict of available eras or it should be able to take in the string. I will work on this but may take a minute. Other requested changes sjust were pushed